### PR TITLE
Enhance blog articles

### PR DESCRIPTION
**Pull request by cveneziani**
- Add images of places
- Add links to Cafés, Coworking spaces, Friends

### CODE REVIEW

File: source/assets/stylesheets/layouts/_blog.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
+.blog {
+  .l-footer {
+    margin-top: 0;
+  }
+}
+
 // .l-blog--article {
```
Comment (flexbox) [inline source/assets/stylesheets/layouts/_blog.scss]: useless comments here

File: source/blog/articles/2015-02-26-london.html.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ READMORE
 
 ### Friends
 
-A good friend and ex-coworker moved there last September. He has invited me several times to come see him. So I taught it was logic to begin my journey by actually visiting him.
+[Kevin](https://kevin.disneur.me/), a good friend and ex-coworker moved there last September. He has invited me several times to come see him. So I taught it was logic to begin my journey by actually visiting him.
```
Comment (flexbox) [inline source/blog/articles/2015-02-26-london.html.md]: Add some SEO juice for friends 

[Kevin, a ruby on rails developer](https://kevin.disneur.me/), good friend and ex-coworker moved there last September.

**Comment by flexbox** [comment]
Review done

**Comment by cveneziani** [comment]
Fix done.

**Comment by flexbox** [comment]
Merge it!

**Comment by cveneziani** [comment]
Thanks!

File: source/blog/articles/2015-02-26-london.html.md (hunk 2 above)
Comment (kdisneur) [inline source/blog/articles/2015-02-26-london.html.md]: Thanks for the SEO juice :kissing_closed_eyes:

**Pull request merged**